beginner_source/translation_transformer.py

```diff
@@ -0,0 +1,399 @@
+"""
+Language Translation with Transformer
+=====================================
+**Author**: `Jayasimha T <https://github.com/iamsimha>`_
+
+This tutorial shows, how to train a translation model from scratch using
+Transformer. We will be using `Multi30k
+dataset <https://github.com/multi30k/dataset>`__ to train a German to
+English translation model.
+
+The data processing and data loading code is taken from an `earlier version
+<https://github.com/pytorch/tutorials/blob/25ece55aac2b3423f8ed06439d0c320bbf40f019/beginner_source/torchtext_translation.py>`__ 
+of tutorial
+
+"""
+
+
+######################################################################
+# Data Processing
+# ---------------
+# 
+# torchtext has utilities for creating datasets that can be easily
+# iterated through for the purposes of creating a language translation
+# model. In this example, we show how to tokenize a raw text sentence,
+# build vocabulary, and numericalize tokens into tensor.
+# 
+# Note: the tokenization in this tutorial requires Spacy We use Spacy
+# because it provides strong support for tokenization in languages other
+# than English. torchtext provides a basic_english tokenizer and supports
+# other tokenizers for English (e.g. Moses) but for language translation -
+# where multiple languages are required - Spacy is your best bet.
+# 
+# To run this tutorial, first install spacy using pip or conda. Next,
+# download the raw data for the English and German Spacy tokenizers:
+# 
+# !python -m spacy download en
```

Comment:
I believe `en` and `de` shortcut are not supported by Spacy 3.0. Since here we use the torchtext API, we don't need the instructions to install spacy. If you really think it's necessary, you can add an URL.

Reply:
If the spacy `en` and `de` models are not downloaded using the above command, we get the below error

`OSError: [E050] Can't find model 'de_core_news_sm'. It doesn't seem to be a Python package or a valid path to a data directory.`

Since these models are not present in Google collab by default, we will always encounter this error. It might be useful to list the commands here so that the error can be debugged quickly.

If you think that only link to https://spacy.io/usage/models is sufficent, I can update this.